### PR TITLE
DOC Add SortPlugin to the docs

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Parent issue
- https://github.com/silverstripe/silverstripe-graphql/issues/565

### CODE REVIEW

File: en/02_Developer_Guides/19_GraphQL/02_working_with_dataobjects/02_query_plugins.md
Comment (ChloeSartorelli) [inline en/02_Developer_Guides/19_GraphQL/02_working_with_dataobjects/02_query_plugins.md]:
```suggestion
In addition, you can use the field sorting plugin ([`SortPlugin`](api:SilverStripe\GraphQL\Schema\Plugin\SortPlugin)) to sort fields that represent `has_many` and `many_many` relationships. To do this, simply add the desired fields to the query, as well as the `sort` argument to these fields. It is also necessary to update the scheme by adding a `sorter` plugin to those fields that need to be sorted. 
```

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: en/02_Developer_Guides/19_GraphQL/02_working_with_dataobjects/02_query_plugins.md
Comment (sabina-talipova) [reply en/02_Developer_Guides/19_GraphQL/02_working_with_dataobjects/02_query_plugins.md]: Updated

**Review by GuySartorelli — APPROVED** [review]
LGTM, will merge after the main PR is merged.

**Pull request merged**
